app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt

```diff
@@ -163,6 +165,12 @@ enum class NavigationItem(
         primaryRoute = NEW_CONVERSATION,
         canonicalRoute = NEW_CONVERSATION,
         content = { NewConversationRouter() }
+    ),
+
+    CallEstablished(
```

Comment:
will there by a screen with CallNotEstablished ? I am wondering because CallEstablished makes me think the call could not be Estabilished

Reply:
Yeah Then I guess changing it to OngoingCallScreen would be better

Reply:
personally, like OngoingCallScreen much more